backend/src/LibExecution/ProgramTypes.fs

```diff
@@ -506,10 +504,10 @@ module TypeDeclaration =
     | Alias of TypeReference
 
     /// `type MyRecord = { a : int; b : string }`
-    | Record of NEList<RecordField>
+    | Record of NEList<RecordField> // NEList
```

Comment:
These are record type definitions. Is the user allowed type:

```fsharp
type MyRecord = {}
```

If we don't allow that, than presumably the type won't be added to the program, which will result with a NameResolution error when it's used. This might be OK? Not sure. Perhaps we'll learn from experience here.